### PR TITLE
Add how to run outer loop libraries tests

### PR DESCRIPTION
**Pull request by karelz**
Add info how to run outer loop libraries tests.
Minor readability improvements.

### CODE REVIEW

**Comment by davidsh** [comment]
I think it would be useful to also add somewhere about how to run Outerloop tests in a PR job, i.e. do "/azp run runtime-libraries outerloop". Of course, we'll need to mention that only folks with write access to the dotnet/runtime repo can do it.

File: docs/workflow/testing/libraries/testing.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ libraries -test /p:WithoutCategories=IgnoreForCI
 
 ## Running tests on the command line
 
-To build tests you need to pass the `-buildtests` flag to build.cmd/sh or if you want to build both src and tests you pass `-buildtests` flag (`libraries -restore -build -buildtests`). Note that you need to specify -restore and -build additionally as those are only implicit if no action is passed in.
+To build tests you need to pass the `-buildtests` flag to `build.cmd/sh` or if you want to build both src and tests you pass `-buildtests` flag (`libraries -restore -build -buildtests`). Note that you need to specify `-restore` and `-build` additionally as those are only implicit if no action is passed in.
```
Comment (stephentoub) [inline docs/workflow/testing/libraries/testing.md]: The first part of the sentence says to pass the -buildtests flag... the second part of the sentence says if you want to build both src and tests to pass the -buildtests flag.  Those are the same.  What am I missing?

File: docs/workflow/testing/libraries/testing.md [hunk 2]
```diff
@@ -47,6 +47,13 @@ To quickly run or debug a single test from the command line, set the XunitMethod
 dotnet build /t:BuildAndTest /p:XunitMethodName={FullyQualifiedNamespace}.{ClassName}.{MethodName}
 ```
 
+#### Running outer loop tests
+
+To run all tests, incl. outer loop tests (which are typically slower or less reliable)
```
Comment (stephentoub) [inline docs/workflow/testing/libraries/testing.md]:
```suggestion
To run all tests, including "outer loop" tests (which are typically slower and in some test suites less reliable but which are more comprehensive):
```

**Review by stephentoub — APPROVED** [review]

File: docs/workflow/testing/libraries/testing.md (hunk 1 above)
Comment (karelz) [reply docs/workflow/testing/libraries/testing.md]: Good point, I have updated the text in my latest commits.

**Comment by karelz** [comment]
@davidsh it is a good idea - not sure where it should go though. @ViktorHofer any idea?

**Comment by stephentoub** [comment]
Conflicts to be addressed.  After that, @karelz, can we get this merged?

**Comment by ViktorHofer** [comment]
Probably here? https://github.com/dotnet/runtime/tree/master/docs/workflow/ci

**Review by ViktorHofer — APPROVED** [review]

**Comment by davidsh** [comment]
@karelz Can you rebase this PR against latest 'master' to unblock the merge conflicts?

**Pull request merged**